index.html

```diff
@@ -1,79 +1,226 @@
+                  </li>
+                  <li class="li">
+                     <a href="https://medium.com/@hopedaphine19" class="assets link"
+                        ><img src="./assets/medium.svg.svg" alt="medium icon"
+                        /></a>
+                  </li>
+                  <li class="li">
+                     <a href="https://github.com/Daphineatim" class="assets link"
+                        ><img src="./assets/github.svg.svg" alt="github icon"
+                        /></a>
+                  </li>
+                  <li class="li">
+                     <a href="https://angel.co/u/hope-daphine" class="assets link"
+                        ><img src="./assets/angellist.svg.svg" alt="angellist icon"
+                        /></a>
+                  </li>
+               </ul>
+            </div>
+         </section>
+         </div>
+         <section id="works">
+          <div class="card_works">
+            <img class="card_works_image" src="./assets/tonic_1.svg" alt="image">
+            <h3 class="card_works_heading">Tonic</h3>
+            <div class="card_works_links">
+              <span>Canopy</span>
+              &middot;
+              <span>Backend Dev</span>               
+              &middot;
+              <span>2015</span>
+           </div>
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+             <div class="card_works_tags">
+               <span>html</span>
+               <span>css</span>
+               <span>javascript</span>
+             </div>
+             <button class="card_works_button">See Project</button>
           </div>
-          <div class="humberger menu"></div>
-          <div class="humberger">
-            <img src="humberger.svg" alt="humberger icon" />
+          <div class="card_works">
+           <img class="card_works_image" src="./assets/multi_post_1.svg" alt="image">
+           <h2 class="card_works_heading">Multi-Post Stories</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
           </div>
-        </nav>
-      </header>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/tonic_2.svg" alt="image">
+           <h2 class="card_works_heading">Tonic</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
```

Comment:
Please ensure that the work section has the margins (top, bottom, left, right) between each card as defined in the design. Also please ensure that the button is as defined in the design.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154817270-56929898-001f-4258-9efa-447d0d4c5f37.png) | ![image](https://user-images.githubusercontent.com/10332499/154817324-ed4d7943-cce2-46d0-b2f1-c91124d8f91c.png)